Core/include/Acts/Vertexing/HelicalTrackLinearizer.hpp

```diff
@@ -103,13 +103,15 @@ class HelicalTrackLinearizer {
   /// @param linPoint Point which defines the Perigee.
   /// @note Transverse plane of the Perigee corresponding to @p linPoint is
   /// parallel to the global x-y plane
+  /// @param perigeeSurface Perigee surface belonging to @p linPoint
```

Comment:
This is maybe a bit much to ask or not sensible at all but I just wanted to mention it here:

By passing both `linPoint` and the `perigeeSurface` we create some redundancy because both of them contain the first 3 entries (i.e., the spatial components) of `linPoint`. Should we maybe introduce a way of constructing a `perigeeSurface` from a 4-Vector and just save the time component as part of the surface? Then, one could retrieve `linPoint` from `perigeeSurface` and we would not have to pass both arguments here.

Reply:
I see your point. We could potentially make something like a 4D perigee surface that is not a purely geometrical object anymore. On the other hand, I'm not sure the extra duplication is something we want.

One alternative to resolve the info redundancy here would be to not pass in the 4D `linPoint`, but literally just the `PerigeeSurface` and a `double time` as a separate argument. That way the information content is the same.

Reply:
This would work, but I would make it very explicit that the `time` is the time component of the Perigee's reference point. We could call it something like `referenceTimePerigee` (or similar) and add an explanatory comment.

Reply:
Changed now. Can you have another look?

Reply:
I think there are some compile problems...